backupTool.py

```diff
@@ -0,0 +1,148 @@
+    'notifications': ['guid', 'handle', 'message', 'timestamp', 'avatar_hash'],
+    'vendors': ['guid', 'ip', 'port', 'signedPubkey'],
+    'moderators': ['guid', 'signedPubkey', 'encryptionKey', 'encryptionSignature', 'bitcoinKey', 'bitcoinSignature', 'handle'],
+    'purchases': ['id', 'title', 'timestamp', 'btc', 'address', 'status', 'thumbnail', 'seller', 'proofSig'],
+    'sales': ['id', 'title', 'timestamp', 'btc', 'address', 'status', 'thumbnail', 'seller'],
+    'dht': ['keyword', 'id', 'value', 'birthday']
+}
+
+def _getDatabase():
+    """Retrieves the OpenBazaar database file."""
+    Database = db.Database()
+    return Database.DATABASE
+
+def silentRemove(filename):
+    """Silently removes a file if it exists."""
+    try:
+        os.remove(filename)
+    except OSError as e:
+        if e.errno != errno.ENOENT: # ENOENT: no such file or directory
+            raise
+
+def _exportDatabaseToCsv(tablesAndColumns):
+    """Reads the database for all given tables and stores them as CSV files."""
+    dbFile = _getDatabase()
+    result = None
+    with lite.connect(dbFile) as dbConnection:
+        dbConnection.text_factory = str
+        cursor = dbConnection.cursor()
+        for table in tablesAndColumns:
+            table_name = table[0]
+            table_columns = ', '.join(table[1])
+            data = cursor.execute("SELECT {0} FROM {1}".format(table_columns, table_name))
+            fileName = 'table_{0}.csv'.format(table_name)
+            filePath = os.path.join('backup', fileName)
+            with open(filePath, 'wb') as f:
+                writer = csv.writer(f)
+                writer.writerow(table[1])
+                writer.writerows(data)
+    return result
+
+def backupFiles(output=None):
+    """Archives OpenBazaar files in a single tar archive."""
+    os.chdir(DATA_FOLDER)
+
+    # Archive files
+    files = os.listdir(DATA_FOLDER)
+    if not output:
+        output = 'backup_{0}.tar.gz'.format(time.strftime('%Y-%m-%d'))
+    silentRemove(output)
+    with tarfile.open(output, 'w:gz') as tar:
+        for f in files:
+            tar.add(f)
+        tar.close()
+    return 'Success'
+
+def exportDatabase(tableList, removePrevious=False):
+    """Exports given tables to the OpenBazaar folder."""
+    os.chdir(DATA_FOLDER)
+
+    # Parse table list
+    tableList = tableList.replace(' ', '').split(',')
+    tablesAndColumns = []
+    for table in tableList:
+        if table in _TABLES:
+            tablesAndColumns.append((table, _TABLES[table]))
+        else:
+            return 'ERROR, Table not found: {0}'.format(table)
+
+    # Remove existing database files and re-make them
+    if removePrevious and os.path.exists('backup'):
+        shutil.rmtree('backup')
+    if not os.path.exists('backup'):
+        os.makedirs('backup')
+    _exportDatabaseToCsv(tablesAndColumns)
+    return 'Success'
```

Comment:
why not simply return `True`, this is an invitation to make a typo, specially that word.